lib/gooddata/models/automated_data_distribution.rb

```diff
@@ -13,7 +13,15 @@ def initialize(project)
     end
 
     def process
-      GoodData::Process[:all, project: project, client: project.client].find { |p| p.type == :dataload }
+      GoodData::Process[:all, project: self.project, client: self.project.client].find do |p|
+        p.type == :dataload
+      end
     end
+
+    def create_output_stage(ads, client_id, output_stage_prefix)
```

Comment:
I think `client_id` and `output_stage_prefix` are not mandatory. So this line can be written like this:
```ruby
def create_output_stage(ads, client_id = nil, output_stage_prefix = nil)
```